src/Raven.Client/Documents/Session/DocumentSessionRevisions.cs

```diff
@@ -18,14 +19,43 @@ public DocumentSessionRevisions(InMemoryDocumentSessionOperations session) : bas
         {
         }
 
-        public List<T> GetFor<T>(string id, int start = 0, int pageSize = 25)
+        public List<T> GetFor<T>(string id, int start = 0, int pageSize = 25, bool metadataOnly = false)
```

Comment:
What is the use of `metadataOnly` here? You aren't _getting_ the metadata here.

Reply:
This is a mistake, should have been removed. I'll fix that.

Reply:
https://github.com/ravendb/ravendb/pull/4538